packages/blockchain-link-types/src/params.ts

```diff
@@ -29,8 +29,9 @@ export interface EstimateFeeParams {
         txsize?: number; // btc transaction size
         from?: string; // eth from
         to?: string; // eth to
-        data?: string; // eth tx data
+        data?: string; // eth tx data, sol tx message
         value?: string; // eth tx amount
+        createsAccount?: boolean; // sol account creation
```

Comment:
I would try to be more specific here and also use `is` as prefix which is nice for flags like this because you do not need to parse the booliness of it from the whole name. e.g. `isTxCreatingTokenAccount`